### DISCUSSION TITLE
apisix start multi port，different port use different route

### DISCUSSION THREAD

**Discussion by kxxu** [apache/apisix#7411] [Q&A]
apisix start multi port，how different port use different route. ex diffrent port use diffrent root route, like 
server {
listen 80;
server_name www.xxx.com;
access_log /data/www/log/80_nginx.log combined;
index index.html index.htm index.php;
include /usr/local/nginx/conf/rewrite/none.conf;
root /data/www/website80;

}
server {
listen 8080;
server_name A.xxx.com;
access_log /data/www/log/8080_nginx.log combined;
index index.html index.htm index.php;
root /data/www/website8080;

}

**Comment by tzssangglass** [comment]
same as: #7077?
